examples/BasicProvider.Runtime/BasicProvider.Runtime.fsproj

```diff
@@ -23,4 +25,14 @@
     </ProjectReference>
   </ItemGroup>
 
+  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
+
+  <Target Name="AfterBuild">
```

Comment:
Please don't add these back, they are unnecessary

Reply:
Documented in https://github.com/fsprojects/FSharp.TypeProviders.SDK/pull/339#issuecomment-716881125